controller/setting_controller.go

```diff
@@ -1541,6 +1541,7 @@ func (info *ClusterInfo) collectVolumesInfo() error {
 		return errors.Wrapf(err, "failed to list Longhorn Volumes")
 	}
 	volumeCount := len(volumesRO)
+	volumeCountV1 := types.GetVolumeCountByBackendStoreDriver(longhorn.BackendStoreDriverTypeV1, volumesRO)
```

Comment:
I think `GetVolumeCountByBackendStoreDriver` is very specific for this usage and would not be used later, suggest adding here as a private function instead.

P.S. If needed, we can move it to a shared package later.

Or add `FilterVolumesByBackendStoreDriver` instead, which would be potentially reusable.. 

```go
volumeCountV1 := len(types.FilterVolumesByBackendStoreDriver(volumes, longhorn.BackendStoreDriverTypeV1))
```

Reply:
I prefer removing the function as there are no other uses for either one at this point, and the logic is straightforward.